### PR TITLE
feat: added WebdriverIO as test framework option

### PR DESCRIPTION
**Pull request by christian-bromann**
This patch adds WebdriverIO as test framework option with a minimal setup that includes page objects.

### CODE REVIEW

**Comment by christian-bromann** [comment]
@clarkdo anything I can do to move this forward?

File: packages/cna-template/template/frameworks/webdriverio/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "test:e2e": "wdio wdio.conf.js"
+  },
+  "devDependencies": {
+    "@wdio/cli": "^6.1.5",
```
Comment (clarkdo) [inline packages/cna-template/template/frameworks/webdriverio/package.json:6]: We can put all related packages to a group in renovate.json

**Comment by christian-bromann** [comment]
@clarkdo thanks for reviewing. I added the wdio packages to `renovate.json`.

**Comment by Atinux** [comment]
Following PR #528 528

**Pull request closed without merging**